class-simple-page-ordering.php

```diff
@@ -199,6 +358,53 @@ public static function admin_head() {
 			);
 		}
 
+		/**
+		 * Modify the row actions for hierarchical post types.
+		 *
+		 * This adds the actions to change the parent/child relationships.
+		 *
+		 * @param array   $actions An array of row action links.
+		 * @param WP_Post $post    The post object.
+		 */
+		public static function page_row_actions( $actions, $post ) {
+			$title         = _draft_or_post_title( $post );
+			$edit_link     = get_edit_post_link( $post->ID, 'raw' );
+			$move_in_link  = add_query_arg(
+				array(
+					'action'    => 'spo-move-in',
+					'spo_nonce' => wp_create_nonce( "simple-page-ordering-nonce-move-{$post->ID}" ),
+					'post_type' => $post->post_type,
+				),
+				$edit_link
+			);
+			$move_out_link = add_query_arg(
+				array(
+					'action'    => 'spo-move-out',
+					'spo_nonce' => wp_create_nonce( "simple-page-ordering-nonce-move-{$post->ID}" ),
+					'post_type' => $post->post_type,
+				),
+				$edit_link
+			);
+
+			$actions['spo-move-in'] = sprintf(
+				'<a href="%s" rel="bookmark" aria-label="%s">%s</a>',
+				esc_url( $move_in_link ),
+				/* translators: %s: Post title. */
+				esc_attr( sprintf( __( 'Move &#8220;%s&#8221; in', 'simple-page-ordering' ), $title ) ),
+				__( '< Move in', 'simple-page-ordering' )
+			);
+
+			$actions['spo-move-out'] = sprintf(
+				'<a href="%s" rel="bookmark" aria-label="%s">%s</a>',
+				esc_url( $move_out_link ),
+				/* translators: %s: Post title. */
+				esc_attr( sprintf( __( 'Move &#8220;%s&#8221; out', 'simple-page-ordering' ), $title ) ),
+				__( 'Move out >', 'simple-page-ordering' )
```

Comment:
I'm not super keen on move in / move out as the text links -- it needs something clearer but also short so it fits the actions toolbar. Suggestions are encouraged, thanks.